lib/std/Build/Module.zig

```diff
@@ -18,6 +18,7 @@ frameworks: std.StringArrayHashMapUnmanaged(LinkFrameworkOptions),
 link_objects: std.ArrayListUnmanaged(LinkObject),
 
 strip: ?bool,
+soname: ?bool,
```

Comment:
Shouldn't this be using the `SOName` type in `src/main.zig`? (Which can be moved to `std.zig.SoName` to enable sharing it.)

Reply:
I've moved `SOName` to `std.zig` and updated this to use it instead of `bool`.